hypertrace-trace-enricher/hypertrace-trace-enricher/src/main/resources/configs/common/application.conf

```diff
@@ -67,7 +67,7 @@ enricher {
       port = 50061
       port = ${?ENTITY_SERVICE_PORT_CONFIG}
     }
-    dependencies = ["DefaultServiceEntityEnricher"]
+    dependencies = ["DefaultServiceEntityEnricher", "EndpointEnricher"]
```

Comment:
@kotharironak on what attributes from the `EndpointEnricher` does the backend enricher depend on?

Reply:
nm. Saw your description later.